### DISCUSSION TITLE
Updating source state attributes

### DISCUSSION THREAD

**Discussion by Lofty-Brambles** [twpayne/chezmoi#4161] [Q&A] [answered]
I've updated a few empty files after adding them. After running `chezmoi apply`, they are still stored in the `local/share/chezmoi` directory with an `empty_` prefix. Is this something I should be worried about? Or can I update this to something?
I've started to migrate to chezmoi after a disastrous time with nuking my setup. Don't want to hand-wire something for my dotfiles ever again. 😭

**Comment by twpayne** [comment, marked as answer]
The `empty_` attribute is only needed if you want chezmoi to create empty files. If the files aren't empty then it is harmless and has no effect. You can remove the attribute with the [`chezmoi chattr` command](https://www.chezmoi.io/reference/commands/chattr/).

**Reply by Lofty-Brambles** [reply]
That was *fast*. Thanks a bunch! And happy holidays to you!
